examples/meson.build

```diff
@@ -1,7 +1,9 @@
-executable ('snap-glib',
-            'snap-glib.c',
-            dependencies: [snapd_glib_dep, gio_dep])
+if get_option ('examples')
```

Comment:
Would this work if the if was moved to the toplevel meson.build, i.e.:
```
if get_option('tests')
  subdir ('tests')
fi
if get_option('examples')
  subdir ('examples')
fi
```

Reply:
You are right, is much simpler that way. Done!